src/libs/feature-previews-config.ts

```diff
@@ -83,21 +83,17 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
   },
   {
     id: ENABLE_AZURE_COLLABORATIVE_WORKFLOW_READERS,
```

Comment:
As per conversation today:
- One feature flag will be shown to end users, the Collaborative Workflows flag
- The ability for manual startup and shutdown will remain behind a feature flag only visible in non-prod environments, so that we can do manual testing before releasing
- Once manual startup and shutdown has been tested sufficiently, it will be released directly - if this is prior to removal of the Collaborative Workflows public flag, it will be gated by the public flag, otherwise it will be directly released via removal of the private feature flag.